molgenis-one-click-importer/src/main/java/org/molgenis/oneclickimporter/service/Impl/EntityServiceImpl.java

```diff
@@ -158,4 +170,28 @@ private String asValidAttributeName(String columnName)
 	{
 		return columnName.replace(" ", "_");
 	}
+
+	private String createNonDuplicateLabel(String label)
```

Comment:
add java doc explaining why this is needed and what is happening , consider moving to service and add test